.golangci.yml

```diff
@@ -0,0 +1,29 @@
+# This file contains configuration options for golangci-lint.
+# https://github.com/golangci/golangci-lint/blob/master/.golangci.reference.yml
+
+run:
+  # Timeout for analysis.
+  timeout: 5m
+
+linters:
+  # Disable all linters to prevent appearing new issues on golangci-lint update.
+  disable-all: true
```

Comment:
I would prefer a disable list instead. It doesn't update itself, and that way we get to know about new linters on update instead of never finding out about new ones.

Reply:
But it's up to you. Let's run with `enable-all`. In future, this can be easily changed.

Reply:
Good point, let's keep `disable-all`.